db/seeds.d/020-provisioning_templates_list.rb

```diff
@@ -53,6 +54,7 @@ def seeded_templates
         { :name => 'RancherOS PXELinux', :source => 'PXELinux/rancheros_pxelinux.erb', :template_kind => kinds[:PXELinux] },
         { :name => 'RancherOS provision', :source => 'provision/rancheros_provision.erb', :template_kind => kinds[:provision] },
         { :name => 'UserData default', :source => 'user_data/userdata_default.erb', :template_kind => kinds[:user_data] },
+        { :name => 'UserData open-vm-tools', :source => 'user_data/userdata_open_vm_tools.erb', :template_kind => kinds[:'cloud-init'] },
```

Comment:
We definitely want to have snapshots for these once merged in core :-)

Reply:
The PR (#5986) is ready, we just need someone to hit the button :-)